### PR TITLE
Component development doc fixes

### PR DESCRIPTION
**Pull request by vipul-sharma20**
- Fix missing import in sample pipeline code ea91ba5
- Remove some stray whitespaces in the doc 6bcb3ac

### CODE REVIEW

File: content/en/docs/pipelines/sdk/component-development.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Below is a summary of the steps involved in creating and using a component:
     component for the Kubeflow Pipelines system.
 1.  Use the Kubeflow Pipelines SDK to load your component, use it in a pipeline and run that pipeline.
 
-The rest of this page gives some explanation about input and output data, 
+The rest of this page gives some explanation about input and output data,
```
Comment (8bitmp3) [inline content/en/docs/pipelines/sdk/component-development.md:33]:
```suggestion
The rest of this tutorial provides explanation about input and output data,
```

File: content/en/docs/pipelines/sdk/component-development.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Below is a summary of the steps involved in creating and using a component:
...
 followed by detailed descriptions of the above steps.
```
Comment (8bitmp3) [inline content/en/docs/pipelines/sdk/component-development.md:34]:
```suggestion
followed by detailed descriptions of the previously-mentioned steps.
```

File: content/en/docs/pipelines/sdk/component-development.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title = "Create Reusable Components"
 description = "A detailed tutorial on creating components that you can use in various pipelines"
 weight = 40
-                    
+
 +++
 {{% alert title="Out of date" color="warning" %}}
 This guide contains outdated information pertaining to Kubeflow 1.0. This guide
```
Comment (8bitmp3) [inline content/en/docs/pipelines/sdk/component-development.md:8]:
```suggestion
This tutorial contains outdated information pertaining to Kubeflow 1.0. This guide
```

Comment (8bitmp3) [reply content/en/docs/pipelines/sdk/component-development.md:8]: Hi @Bobgy @IronPan @neuromage @Ark-kun @numerology This tutorial has the out-of-date banner fyi cc @RFMVasconcelos

Comment (8bitmp3) [reply content/en/docs/pipelines/sdk/component-development.md:8]: Line 12: "page" -> "tutorial"

**Comment by Bobgy** [comment]
/assign @chensun @joeliedtke 
For review

**Pull request merged**